types/assertions.d.cts

```diff
@@ -125,12 +125,14 @@ export type Assertions = {
 	truthy: TruthyAssertion;
 };
 
+type Falsy = false | 0 | -0 | 0n | '' | null | undefined;
```

Comment:
I don't think `-0` is a unique type in TS.

Reply:
You're right about that! It just resolves to 0, so was basically a no-op.